crypto/evp/evp_lib.c

```diff
@@ -281,7 +281,7 @@ int EVP_CIPHER_block_size(const EVP_CIPHER *cipher)
     params[0] = OSSL_PARAM_construct_size_t(OSSL_CIPHER_PARAM_BLOCK_SIZE, &v);
     ok = evp_do_ciph_getparams(cipher, params);
 
-    return ok != 0 ? (int)v : EVP_CTRL_RET_UNSUPPORTED;
+    return ok != 0 ? (int)v : -1;
```

Comment:
I guess these ones are ok.. In 1_1_1 I guess it always returned something non negative.

Reply:
Yeah, I take the common pattern to follow from 1.1.1's `EVP_CIPHER_CTX_iv_length()`